vissl/trainer/trainer_main.py

```diff
@@ -185,7 +187,7 @@ def train(self):
                         logging.info("CUDA cache cleared")
                     task = train_step_fn(task)
                     iteration_num += 1
```

Comment:
we can update the iteration property of the storage from line 175 above in this module :)

Reply:
I intended update it here on line 190.
```
task.local_iteration_num = task.event_storage.iter = iteration_num
```

I think iteration being the same on all lines is the expected behavior for `tests/test_tasks.py`, but I noticed that [in this PR](https://github.com/MLH-Fellowship/vissl/pull/8#issuecomment-787131794) we had the following output where the iteration is `1` and not `0`.
```
{"batch_time": 1132.0, "epoch": 0.0, "eta": 1.1322011947631836, "iteration": 0, "loss": 3.00722, "lr": 0.00234, "peak_mem_used": -1.0, "rank": 0.0}
{"batch_time": 1014.0, "epoch": 0.0, "eta": 2.0299949645996094, "iteration": 0, "loss": 6.38364, "lr": 0.00047, "peak_mem_used": -1.0, "rank": 0.0}
{"batch_time": 1936.0, "epoch": 0.0, "eta": 1.9367702007293701, "iteration": 0, "loss": 1.3317, "lr": 0.00234, "peak_mem_used": -1.0, "rank": 0.0}
{"batch_time": 526.0, "epoch": 0.0, "eta": 0.526491641998291, "iteration": 0, "loss": 1.13592, "lr": 0.00234, "peak_mem_used": -1.0, "rank": 0.0}
```

Is there a different/additional place in the code where we should update the iteration number?